### DISCUSSION TITLE
No correct pre-phone number for china user when register 2FA !!!

### DISCUSSION THREAD

**Discussion by pengandzhang** [community/community#66448] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I can not register my phone number to accept txt mesg when I register 2FA , due to not providing china number +86 .

It will make me that can not  sign in my account. 
I need any help to solve this problem!  thks!

**Comment by hafiz-muhammad** [comment]
You can use an authenticator app as an alternative 2FA method.
